src/graphql/node/control.rs

```diff
@@ -39,172 +38,230 @@ impl NodeControlMutation {
         }
     }
 
-    // TODO: Apply node issue #251
+    /// Applies the draft configuration to the node with the given ID.
+    ///
+    /// This function updates the node's `name` with `name_draft`, `profile` with `profile_draft`,
+    /// and `config` values of agents with their `draft` values.
+    ///
+    /// Success is returned as long as the database update is successful, regardless of the outcome
```

Comment:
As I commented, it would be more conventional to start with `Returns` as the first line starts with `Applies`. I attached an example.

<img width="821" alt="스크린샷 2024-09-02 오후 8 48 09" src="https://github.com/user-attachments/assets/f27aefff-8bd1-4ea3-99fe-0bedd7ab82e2">